src/core_plugins/kibana/common/field_formats/types/boolean.js

```diff
@@ -0,0 +1,29 @@
+import { asPrettyString } from '../../utils/as_pretty_string';
```

Comment:
Were any changes made to the types that are showing up as additions and not renames in the diff?